.github/workflows/ci.yaml

```diff
@@ -0,0 +1,70 @@
+#
+# Copyright (c) 2020 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+# Che-Theia workflow
```

Comment:
FYI copy paste from this line and the next one should be adapted

Reply:
sure. Missed that.